packages/@uppy/locales/src/fr_FR.js

```diff
@@ -86,14 +92,19 @@ fr_FR.strings = {
   saveChanges: 'Sauvegarder les modifications',
   selectAllFilesFromFolderNamed: 'Sélectionner tous les fichiers du dossier %{name}',
   selectFileNamed: 'Sélectionner le fichier %{name}',
+  selectSourceDescription: 'Afin de réaliser une vidéo de votre écran, veuillez autoriser l\'accès à ce site.',
+  selectSourceTitle: 'Veuillez autoriser l\'accès à votre écran',
   selectX: {
     '0': 'Sélectionner %{smart_count}',
     '1': 'Sélectionner %{smart_count}',
     '2': 'Sélectionner %{smart_count}'
   },
-  smile: 'Souris!',
+  smile: 'Souriez !',
```

Comment:
Space before exclamationmark intended?

Reply:
Yes, this is part of the French typographic rules

Reply:
http://unicode.org/udhr/n/notes_fra.html